config/default/common/config/metadata/layers/modis/VegetationIndices.md

```diff
@@ -1,53 +1,53 @@
 
-References: [MOD13Q4N dataset doi:10.5067/MODIS/MOD13Q4N.NRT.061](https://doi.org/10.5067/MODIS/MOD13Q4N.NRT.061); [NASA Earth Observatory - Measuring Vegetation](https://earthobservatory.nasa.gov/Features/MeasuringVegetation/measuring_vegetation_4.php); [NASA Earthdata - New Vegetation Indices and Surface Reflectance Products Available from LANCE](https://earthdata.nasa.gov/learn/articles/new-vegetation-indices-and-surface-reflectance-products-available-from-lance); [NASA NEO - Vegetation Index](https://neo.sci.gsfc.nasa.gov/view.php?datasetId=MOD_NDVI_M)
-
+References: MOD13Q4N [doi:10.5067/MODIS/MOD13Q4N.NRT.061](https://doi.org/10.5067/MODIS/MOD13Q4N.NRT.061); [NASA Earth Observatory - Measuring Vegetation](https://earthobservatory.nasa.gov/Features/MeasuringVegetation/measuring_vegetation_4.php); [NASA Earthdata - New Vegetation Indices and Surface Reflectance Products Available from LANCE](https://earthdata.nasa.gov/learn/articles/new-vegetation-indices-and-surface-reflectance-products-available-from-lance); [NASA NEO - Vegetation Index](https://neo.sci.gsfc.nasa.gov/view.php?datasetId=MOD_NDVI_M)
 
 ### MODIS (Terra/Aqua) Vegetation Index (Level 3, 16-Day)
-Temporal coverage: 4 March 2000 - present (Terra); July 2002 - present (Aqua)
+Temporal coverage: 4 March 2000 - present (Terra); 4 July 2002 - present (Aqua)
 
-The Terra/Aqua Moderate Resolution Imaging Spectroradiometer (MODIS) Vegetation Indices (MOD13Q1/MYD13Q1) Version 6 data are generated every 16 days at 250 meter (m) spatial resolution as a Level 3 product. The MOD13Q1/MYD13Q1 product provides Normalized Difference Vegetation Index (NDVI) which is referred to as the continuity index to the existing National Oceanic and Atmospheric Administration-Advanced Very High Resolution Radiometer (NOAA-AVHRR) derived NDVI. The algorithm chooses the best available pixel value from all the acquisitions from the 16-day period. The criteria used is low clouds, low view angle, and the highest NDVI value.
+The Vegetation Index (L3, 16-Day) layers are created from the Terra/Aqua Moderate Resolution Imaging Spectroradiometer (MODIS) Vegetation Indices (MOD13Q1/MYD13Q1) data which are generated every 16 days at 250 meter (m) spatial resolution as a Level 3 product. The MOD13Q1/MYD13Q1 products provide Normalized Difference Vegetation Index (NDVI) which is referred to as the continuity index to the existing National Oceanic and Atmospheric Administration-Advanced Very High Resolution Radiometer (NOAA-AVHRR) derived NDVI. The algorithm chooses the best available pixel value from all the acquisitions from the 16-day period. The criteria used is low clouds, low view angle, and the highest NDVI value.
```

Comment:
(MOD13Q1/MYD13Q1) data which `is` generated

Reply:
I feel like "are" is ok, so i'm leaving it ;)